ci/kokoro/windows/build-32.bat

```diff
@@ -16,18 +16,18 @@ REM Install Bazelisk.
 @echo %date% %time%
 @cd github\google-cloud-cpp
 @powershell -exec bypass ci\kokoro\windows\install-bazelisk.ps1
-@if NOT ERRORLEVEL 0 exit /b 1
+@if %ERRORLEVEL% neq 0 exit /b 1
```

Comment:
I would suggest sticking with `@if ERRORLEVEL 1` (et seq).  `%ERRORLEVEL%` first looks for an environment variable of that name, which, if it exists, is probably not the error level of the last command (https://devblogs.microsoft.com/oldnewthing/20080926-00/?p=20743).

Reply:
TIL. Done.